com.unity.ml-agents/Runtime/Sensors/StackingSensor.cs

```diff
@@ -233,7 +227,7 @@ internal byte[] CreateEmptyPNG()
         }
 
         /// <summary>
-        /// Constrct stacked CompressedChannelMapping.
+        /// Construct stacked CompressedChannelMapping.
         /// </summary>
         internal int[] ConstructStackedCompressedChannelMapping(ISensor wrappedSenesor)
```

Comment:
Thought a bit about moving this to the CompressionSpec constructor, don't think it's necessary here though.
But if one day we add more compression types it might make sense to move this so we can do something like CompressionSpec(PNG, num_layer=4, stacking=2) and have it automatically setup for different compression.

Reply:
Yeah, I think we can move more utility methods into CompressionSpec (even if they stay `internal static`). I think I'll leave this here for now, but we can move it later if it makes sense (and there's another use case)